app/views/reports/templates/scinote_template/header.html.erb

```diff
@@ -22,10 +22,10 @@
 </style>
 <div class="print-header">
   <div class="logo-img">
-    <% if logo == 'logo.png' %>
+    <% if logo == 'scinote_logo.svg' %>
       <%= image_tag wicked_pdf_asset_base64(logo) %>
-    <%else%>
-      <%= wicked_pdf_image_tag logo %>
```

Comment:
is it really needed?

Reply:
Yes logo in this case is base64 encoded image. `wicked_pdf_asset_base64` only works with images from the asset pipeline not S3